src/main/java/net/masterthought/cucumber/json/Result.java

```diff
@@ -37,6 +37,10 @@ public String getErrorMessage() {
     }
 
     public final String getErrorMessageTitle() {
-        return errorMessage.split("[\\r\\n]+")[0];
+        try {
+            return errorMessage.split("[\\r\\n]+")[0];
+        } catch(ArrayIndexOutOfBoundsException e) {
```

Comment:
definitely not this way - this is more like cure for the result but I prefer code that prevent this situation